### ISSUE TITLE
add explicit option to specify the DB backend

### ISSUE THREAD

**Issue by tomasohara** [heartexlabs/label-studio#1660]
**Is your feature request related to a problem? Please describe.**
When switching between sqlite3 and postgresql backend, there is no command-line option to indicate that postgresql should be used. Instead this is inferred from the various POSTGRE_XYZ environment variables (see https://labelstud.io/guide/storedata.html#PostgreSQL-database).

Note that several times I made the following mistake:
```
$ label-studio start --database postgresql
...
^C

$ file postgresql
postgresql: SQLite 3.x database, last written using SQLite version 3035004
```

**Describe the solution you'd like**
Add an option like --db-backend to explicitly specify which one should be used.

**Describe alternatives you've considered**
One alternative is to use two separate terminal windows, so that the environment settings don't have to be changed to do the switch.

**Additional context**
The idea came to mind when diagnosing issue #1658: several times I had to switch between sqlite3 and postgresql.

**Comment by makseq** [comment]
This little mess with DB naming was fixed in 1.4.

**Comment by hashdashd** [comment]
@makseq @tomasohara don't suppose you guys would mind taking a look at my stack overflow question on this. I can't seem to get a postgresql back end working.
https://stackoverflow.com/questions/72093304/creating-a-postgresql-database-back-end-for-a-new-label-studio-project
cheers
